sklearn/preprocessing/_target_encoder.py

```diff
@@ -322,35 +381,142 @@ def _fit_encodings_all(self, X, y):
             dtype=np.int64,
             count=len(self.categories_),
         )
+        if self.target_type_ == "multiclass":
+            encodings = self._learn_multiclass_encodings(
+                X_ordinal,
+                y,
+                n_categories,
+                self.target_mean_,
+            )
+        else:
+            encodings = self._learn_binary_or_continuous_encoding(
+                X_ordinal,
+                y,
+                n_categories,
+                self.target_mean_,
+            )
+        self.encodings_ = encodings
+
+        return X_ordinal, X_known_mask, y, n_categories
+
+    def _learn_binary_or_continuous_encoding(
```

Comment:
for the sake of consistency, I would rename this to: `_fit_encoding_binary_or_continuous` and rename `_learn_multiclass_encodings` similarly.